Ugizwenayo-Divine/FoodStuffPriceCompare-App
# setting up coveralls

#### What does this PR do?
- setting up the coveralls for the project
#### Description of Task to be completed?
- installed all needed packages(coveralls,nyc and mocha-lcov-reporter)
- added the git repo to coveralls
- created the .coveralls.yml file
- added the repo token from coveralls to the .coveralls.yml file
#### How should this be manually tested?
- After cloning the project repo, run npm i inside the cloned repo folder
- change in cloned repo
- git fetch
- git checkout ch-setup-coveralls
- npm i
- you must see the coverage badge
#### Any background context you want to provide?
- you must have node installed locally
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
